src/services/CertificateGenerationService.ts

```diff
@@ -164,23 +128,13 @@ class CertificateGenerationService {
 		return shouldTranslateTestResult;
 	}
 
-	/**
-	 * Determine if abandoned certificates should be generated for the given test result
-	 */
-	public shouldGenerateAbandonedCerts(): boolean {
-		if (!CertificateGenerationService.flags.abandonedCerts.enabled) {
-			console.warn(`Unable to generate abandoned certificates: global abandoned certificates flag disabled.`);
-			return false;
-		}
-		return true;
-	}
-
 	/**
 	 * Method to check if Welsh translation is enabled.
 	 * @returns boolean
 	 */
 	public isGlobalWelshFlagEnabled(): boolean {
-		if (!CertificateGenerationService.flags.welshTranslation.enabled) {
+		console.log(CertificateRequestProcessor.flags.welshTranslation.enabled);
```

Comment:
Tiny comment, but do we want to add a message to this, otherwise just a 'true' / 'false' will print out and it's not overly clear?

e.g. `console.log('isGlobalWelshFlagEnabled', CertificateRequestProcessor.flags.welshTranslation.enabled);`

Reply:
This has now been done